packages/metal-component/src/Component.js

```diff
@@ -439,8 +439,8 @@ class Component extends EventEmitter {
 		}
 		const instance = new Ctor(config, false);
 
-		if (window.__METAL_DEV_TOOLS_HOOK__) {
-			window.__METAL_DEV_TOOLS_HOOK__(instance);
+		if (window.__METAL_DEV_TOOLS_HOOK__ && window.__METAL_DEV_TOOLS_HOOK__.addRoot) {
+			window.__METAL_DEV_TOOLS_HOOK__.addRoot(instance);
```

Comment:
Yeah you are totally right about having the backwards compatibility. Ill update this PR.

As far as moving devtools under the metal repo, I think it should wait until it is adopted a bit more so we can get to a more stable release of it. Im sure there are still plenty of issues I haven't encountered yet.